drivers/scsiutil.py

```diff
@@ -361,6 +361,7 @@ def _genArrayIdentifier(dev):
     try:
```

Comment:
I think this is only used by the obsolete scripts below, if so it can be safely removed when the scripts are deleted.